dotCMS/src/main/java/com/dotmarketing/common/db/DotDatabaseMetaData.java

```diff
@@ -0,0 +1,313 @@
+package com.dotmarketing.common.db;
+
+import com.dotcms.util.CloseUtils;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.Logger;
+import com.google.common.collect.ImmutableList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.ImmutableList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)